packs/astral/config.schema.yaml

```diff
@@ -0,0 +1,11 @@
+---
+  latitude:
+    description: "Latitude of the location"
+    type: "integer"
```

Comment:
Will probably need to use `number` here because integer doesn't allow decimal values iirc.